index.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
+
+const fsCallback = require("./lib/files-callBack.js");
+const fsPromise = require("./lib/file-promise.js"), write = require("./lib/write");
+const file = process.argv.slice(2)[0];
+
+
+
+/** this is just a function to run 
+ * callBack with two parameters */
+
+function myTestcallBack(err, data) {
+  console.log("hi!")
+}
+/**use callBack read write read
+ * read l
+ * @param file
+ * @param callBack
+ * write
+ * change data field value 
+ * @param file
+ * @param data
+ * @param callBack
+ * read ll
+ * @param file
+ * @param callBack
+ */
+const useCallbacks = cb => {
+  fsCallback.read(file, (err, data) => {
+    if (err) {
+      console.error(err);
+    } else
+      data.lastName = "Callback";
+    fsCallback.write(file, data, (err, result) => {
+      if (err) {
+        console.error(err);
+      } else {
+        fsCallback.read(file, (err, afterData) => {
+          cb(afterData)
+        })
+      }
+
+    })
+  })
+}
+
+/** 
+ * use Promise
+ *  read l
+ * @param file
+ * change data field value
+ * @return data
+ * write
+ * @param file
+ * @param obj then 
+ * read ll
+ * @param file
+ */
+const usePromise = () => {
+  return fsPromise
+    .read(file)
+    .then(data => {
+      data.lastName = "promise";
+      return data;
+
+    })
+    .then(obj => write.write(file, obj))
```

Comment:
This line is causing your code to break: 
<img width="949" alt="Screen Shot 2019-10-27 at 9 37 56 PM" src="https://user-images.githubusercontent.com/22336391/67652463-4ab2e680-f902-11e9-890f-df33fc2bf3cc.png">